<ui version="4.0" >
  <author/>
  <comment/>
  <exportmacro/>
  <class>FindIconDialog</class>
  <widget class="QDialog" name="FindIconDialog" >
    <property name="objectName" >
      <string notr="true" >FindIconDialog</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>539</width>
        <height>468</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Find Icon</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="objectName" >
        <string notr="true" />
      </property>
      <property name="margin" >
        <number>9</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QRadioButton" name="m_specify_file_input" >
          <property name="objectName" >
            <string notr="true" >m_specify_file_input</string>
          </property>
          <property name="geometry" >
            <rect>
              <x>9</x>
              <y>9</y>
              <width>521</width>
              <height>18</height>
            </rect>
          </property>
          <property name="text" >
            <string>Specify image file</string>
          </property>
          <property name="checked" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="m_specify_file_box" >
          <property name="objectName" >
            <string notr="true" >m_specify_file_box</string>
          </property>
          <property name="geometry" >
            <rect>
              <x>9</x>
              <y>33</y>
              <width>521</width>
              <height>182</height>
            </rect>
          </property>
          <property name="title" >
            <string>File</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="objectName" >
              <string notr="true" />
            </property>
            <property name="margin" >
              <number>9</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <layout class="QHBoxLayout" >
                <property name="objectName" >
                  <string notr="true" />
                </property>
                <property name="margin" >
                  <number>1</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item>
                  <widget class="QLabel" name="label_2" >
                    <property name="objectName" >
                      <string notr="true" >label_2</string>
                    </property>
                    <property name="geometry" >
                      <rect>
                        <x>1</x>
                        <y>1</y>
                        <width>59</width>
                        <height>23</height>
                      </rect>
                    </property>
                    <property name="frameShape" >
                      <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow" >
                      <enum>QFrame::Plain</enum>
                    </property>
                    <property name="text" >
                      <string>Directory:</string>
                    </property>
                    <property name="textFormat" >
                      <enum>Qt::AutoText</enum>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="m_file_dir_input" >
                    <property name="objectName" >
                      <string notr="true" >m_file_dir_input</string>
                    </property>
                    <property name="geometry" >
                      <rect>
                        <x>66</x>
                        <y>3</y>
                        <width>351</width>
                        <height>19</height>
                      </rect>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="echoMode" >
                      <enum>QLineEdit::Normal</enum>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="m_file_dir_browse" >
                    <property name="objectName" >
                      <string notr="true" >m_file_dir_browse</string>
                    </property>
                    <property name="geometry" >
                      <rect>
                        <x>423</x>
                        <y>1</y>
                        <width>75</width>
                        <height>23</height>
                      </rect>
                    </property>
                    <property name="text" >
                      <string>Browse</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QLabel" name="label_4" >
                <property name="objectName" >
                  <string notr="true" >label_4</string>
                </property>
                <property name="geometry" >
                  <rect>
                    <x>11</x>
                    <y>51</y>
                    <width>499</width>
                    <height>13</height>
                  </rect>
                </property>
                <property name="frameShape" >
                  <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow" >
                  <enum>QFrame::Plain</enum>
                </property>
                <property name="text" >
                  <string>Image files:</string>
                </property>
                <property name="textFormat" >
                  <enum>Qt::AutoText</enum>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QListWidget" name="m_file_image_list" >
                <property name="objectName" >
                  <string notr="true" >m_file_image_list</string>
                </property>
                <property name="geometry" >
                  <rect>
                    <x>11</x>
                    <y>70</y>
                    <width>499</width>
                    <height>101</height>
                  </rect>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QRadioButton" name="m_specify_resource_input" >
          <property name="objectName" >
            <string notr="true" >m_specify_resource_input</string>
          </property>
          <property name="geometry" >
            <rect>
              <x>9</x>
              <y>221</y>
              <width>521</width>
              <height>18</height>
            </rect>
          </property>
          <property name="text" >
            <string>Specify resource</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="m_specify_resource_box" >
          <property name="objectName" >
            <string notr="true" >m_specify_resource_box</string>
          </property>
          <property name="enabled" >
            <bool>false</bool>
          </property>
          <property name="geometry" >
            <rect>
              <x>9</x>
              <y>245</y>
              <width>521</width>
              <height>181</height>
            </rect>
          </property>
          <property name="title" >
            <string>Resource</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="objectName" >
              <string notr="true" />
            </property>
            <property name="margin" >
              <number>9</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <layout class="QHBoxLayout" >
                <property name="objectName" >
                  <string notr="true" />
                </property>
                <property name="margin" >
                  <number>1</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item>
                  <widget class="QLabel" name="label" >
                    <property name="objectName" >
                      <string notr="true" >label</string>
                    </property>
                    <property name="geometry" >
                      <rect>
                        <x>1</x>
                        <y>1</y>
                        <width>79</width>
                        <height>21</height>
                      </rect>
                    </property>
                    <property name="frameShape" >
                      <enum>QFrame::NoFrame</enum>
                    </property>
                    <property name="frameShadow" >
                      <enum>QFrame::Plain</enum>
                    </property>
                    <property name="text" >
                      <string>Resource file:</string>
                    </property>
                    <property name="textFormat" >
                      <enum>Qt::AutoText</enum>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QComboBox" name="m_resource_combo" >
                    <property name="objectName" >
                      <string notr="true" >m_resource_combo</string>
                    </property>
                    <property name="geometry" >
                      <rect>
                        <x>86</x>
                        <y>1</y>
                        <width>412</width>
                        <height>21</height>
                      </rect>
                    </property>
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>7</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QLabel" name="label_3" >
                <property name="objectName" >
                  <string notr="true" >label_3</string>
                </property>
                <property name="geometry" >
                  <rect>
                    <x>11</x>
                    <y>49</y>
                    <width>499</width>
                    <height>13</height>
                  </rect>
                </property>
                <property name="frameShape" >
                  <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow" >
                  <enum>QFrame::Plain</enum>
                </property>
                <property name="text" >
                  <string>Resources:</string>
                </property>
                <property name="textFormat" >
                  <enum>Qt::AutoText</enum>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTreeView" name="m_resource_tree" >
                <property name="objectName" >
                  <string notr="true" >m_resource_tree</string>
                </property>
                <property name="geometry" >
                  <rect>
                    <x>11</x>
                    <y>68</y>
                    <width>499</width>
                    <height>102</height>
                  </rect>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="objectName" >
            <string notr="true" />
          </property>
          <property name="margin" >
            <number>1</number>
          </property>
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <spacer>
              <property name="objectName" >
                <string notr="true" >horizontalSpacer</string>
              </property>
              <property name="geometry" >
                <rect>
                  <x>1</x>
                  <y>1</y>
                  <width>355</width>
                  <height>25</height>
                </rect>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
                <size>
                  <width>40</width>
                  <height>20</height>
                </size>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="m_cancel_button" >
              <property name="objectName" >
                <string notr="true" >m_cancel_button</string>
              </property>
              <property name="geometry" >
                <rect>
                  <x>362</x>
                  <y>2</y>
                  <width>75</width>
                  <height>23</height>
                </rect>
              </property>
              <property name="text" >
                <string>Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_ok_button" >
              <property name="objectName" >
                <string notr="true" >m_ok_button</string>
              </property>
              <property name="enabled" >
                <bool>false</bool>
              </property>
              <property name="geometry" >
                <rect>
                  <x>443</x>
                  <y>1</y>
                  <width>77</width>
                  <height>25</height>
                </rect>
              </property>
              <property name="text" >
                <string>Ok</string>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <pixmapfunction/>
  <resources/>
</ui>
